core-plugin/src/com/google/gct/idea/debugger/CloudLineBreakpointType.java

```diff
@@ -166,6 +169,8 @@ public CloudLineBreakpointProperties createBreakpointProperties(@NotNull Virtual
   @NotNull
   @Override
   public Breakpoint createJavaBreakpoint(Project project, XBreakpoint breakpoint) {
+    UsageTrackerService.getInstance()
+        .trackEvent(GctTracking.CATEGORY, GctTracking.CLOUD_DEBUGGER, "create.breakpoint", null);
```

Comment:
turns out this is wrong. This method is called under a variety of conditions, need to look deeper in to it.